src/Integration/MefServices/SharedBindingSuggestionService.cs

```diff
@@ -88,6 +91,18 @@ private void OnActiveSolutionChanged(object sender, ActiveSolutionChangedEventAr
             {
                 Suggest();
             }
+            else
+            {
+                suggestSharedBindingGoldBar.Close();
+            }
+        }
+        
+        private void OnActiveSolutionBindingChanged(object sender, ActiveSolutionBindingEventArgs e)
```

Comment:
Ideally this should be split into a different PR

Reply:
Created a separate PR for the close notification on solution close https://github.com/SonarSource/sonarlint-visualstudio/pull/5781